docs/oba-release-sop.md

```diff
@@ -43,44 +43,46 @@ sh run.sh make prepare_release -B
 
 ### Check and package the release artefacts for OBA
 
-- [ ] You should also check in Protege if the new terms you just added look fine.
-- [ ] Open in Protege some of the OBA release artefacts and check for any potential errors.
+- You should also check in Protege if the new terms you just added look fine.
+- Open in Protege some of the OBA release artefacts and check for any potential errors.
 For example, check if there are any unsatisfiable classes in `oba.obo`.
 
 
-For background information on release artefacts, see
-* [OWL, OBO, JSON? Base, simple, full, basic? What should you use, and why?](https://oboacademy.github.io/obook/explanation/owl-format-variants/)
-* [Release artefacts](https://oboacademy.github.io/obook/reference/release-artefacts/)
-
 GitHub imposes some size constraints. The size of all the OBA artefacts exceeds the GitHub imposed size limit, but you can attach all the release artefacts as binary files to the release product.
```

Comment:
Isn't this same point addressed below? In "NOTE:..."

Reply:
Yes, this sentence belongs to the next section. I am going to move it there.